src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -3787,6 +3787,54 @@ namespace Google.Example.Library.V1
         /// <summary>
         /// Test optional flattening parameters of all types
         /// </summary>
+        /// <param name="callSettings">
+        /// If not null, applies overrides to this RPC call.
+        /// </param>
+        /// <returns>
+        /// A Task containing the RPC response.
+        /// </returns>
+        public virtual Task<TestOptionalRequiredFlatteningParamsResponse> TestOptionalRequiredFlatteningParamsAsync(
+            CallSettings callSettings = null) => TestOptionalRequiredFlatteningParamsAsync(
+                new TestOptionalRequiredFlatteningParamsRequest
+                {
+
+                },
+                callSettings);
+
+        /// <summary>
+        /// Test optional flattening parameters of all types
+        /// </summary>
+        /// <param name="cancellationToken">
+        /// A <see cref="CancellationToken"/> to use for this RPC.
+        /// </param>
+        /// <returns>
+        /// A Task containing the RPC response.
+        /// </returns>
+        public virtual Task<TestOptionalRequiredFlatteningParamsResponse> TestOptionalRequiredFlatteningParamsAsync(
+            CancellationToken cancellationToken) => TestOptionalRequiredFlatteningParamsAsync(
+
```

Comment:
I'm assuming it's reasonably tricky to remove this empty line, and the one from the method above?